Fitness/Views/GymFacilitiesView.swift

```diff
@@ -0,0 +1,483 @@
+//
+//  GymFacilitiesView.swift
+//  Fitness
+//
+//  Created by Ji Hwan Seung on 4/28/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+
+enum FacilityName {
+    case helenNewman
+    case teagle
+    case noyes
+    case appel
+}
+
+class FacilitiesGestureRecognizer: UITapGestureRecognizer {
+    var title = String()
+}
+
+class GymFacilitiesTableView: UIView, UITableViewDataSource, UITableViewDelegate {
+    
+    let helenNewmanFacilities: [FacilityData] = [
+        FacilityData(facility: "BASKETBALL COURT", isDropped: false, data: [
+            ("10 AM - 12 AM", "MON"),
```

Comment:
When will this data be from backend?

Reply:
Probably not in the near future. It's located in a very wonky place, and given that it's static data (that's also little), it's not a top priority from backend perspective